done

heyam09 opened the pull request:
## Before submitting, make sure your project meets all of these requirements!

### Checklist
**Use an `[x]` to mark completed tasks!**
- [ ] A paragraph introducing yourself
- [ ] A paragraph about your dreams & goals!
- [ ] An unordered list of your interests
- [ ] A photo of yourself OR an avatar that represents yourself
- [ ] Three other images
- [ ] A link to your Github and any other professional social media (Twitter, LinkedIn)

epq commented:
Great work! :confetti_ball:

epq commented on index.html:
```diff
@@ -0,0 +1,44 @@
+<body>
+	<h1>My profile</h1>
+<p>
+	My name is Heyam, I am from Syria, and I am 20 <br>years old. I live in Canada with my family which is my mom, Dad, two brothers, and little sister since 2016. I have graduated from high school before I come to Canada, I like to play Piano and read books in my free time. I am very social person and I very like to spend time with friends and family. Also, I am always interesting to learn new thin ...
```
Why did you add a `<br>` here in the middle of your sentence?

heyam09 commented:
Hello,

 The <br> was by mistake I wanted  to put <br> for all of the lines, but
 then I changed my mind and removed them, but forgot to remove this one.

On Sun, Jul 28, 2019 at 10:57 PM Elizabeth Chan <notifications@github.com>
wrote:

> *@epq* commented on this pull request.
> ------------------------------
>
> In index.html
> <https://github.com/hackyourfuturecanada/Week2/pull/13#discussion_r308041127>
> :
>
> > @@ -0,0 +1,44 @@
> +<!DOCTYPE html>
> +<html>
> +<head>
> +	<title>week2</title>
> +</head>
> +<body>
> +	<h1>My profile</h1>
> +<p>
> +	My name is Heyam, I am from Syria, and I am 20 <br>years old. I live in Canada with my family which is my mom, Dad, two brothers, and little sister since 2016. I have graduated from high school before I come to Canada, I like to play Piano and read books in my free time. I am very social person and I very like to spend time with friends and family. Also, I am always interesting to learn new things and trying to work hard in order to reach my goals and dreams.
>
> Why did you add a <br> here in the middle of your sentence?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/hackyourfuturecanada/Week2/pull/13?email_source=notifications&email_token=AMOVXW44JMQ34V37RZ4PEJDQBZMANA5CNFSM4IHNE32KYY3PNVWWK3TUL52HS4DFWFIHK3DMKJSXC5LFON2FEZLWNFSXPKTDN5WW2ZLOORPWSZGOB7ZHZHQ#pullrequestreview-267549854>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AMOVXW4YMAZMAJFHVTDLY4LQBZMANANCNFSM4IHNE32A>
> .
>
